### DISCUSSION TITLE
[i18n] How to translate metadata from themeConfig

### DISCUSSION THREAD

**Discussion by mariuszkrzaczkowski** [facebook/docusaurus#8608] [Q&A]
I'm trying to translate text in metadata for keywords and description and failing.
Does anyone know how this can be done?

I tried
- i18n\pl\code.json
- i18n\pl\docusaurus-theme-classic\navbar.json

my example
```
themeConfig: {
	metadata: [
		{ name: 'keywords', content: 'yetiforce,help,doc,docs,documentation,guides,wiki,knowledge' },
		{ name: 'description', content: 'Official YetiForce Open Source System documentation / guide / knowledge' }
	],
],
```

**Comment by slorber** [comment]
There are some things that we, unfortunately, can't translate for now.

Please report them here if not already mentioned: https://github.com/facebook/docusaurus/issues/4542
